website/src/App.css

```diff
@@ -12,14 +12,14 @@
   background: linear-gradient(135deg, rgba(210, 31, 241, 1) 0%, rgba(130, 51, 232, 1) 49%, rgba(0, 212, 255, 1) 100%);
 }
 
-.protected-view-loading svg {
-  width: 50px !important;
-  height: auto !important;
-  color: white !important;
+#root .protected-view-loading svg {
```

Comment:
Good work, selector specificity is way better than the important tag

Reply:
Thanks Tobias!